server/plugin.go

```diff
@@ -287,6 +291,7 @@ func (p *Plugin) handleComplete(w http.ResponseWriter, r *http.Request) {
 	}
 
 	issue, err := p.listManager.CompleteIssue(userID, completeRequest.ID)
```

Comment:
What do you think about adding the list to the return value of CompleteIssue, and use it to only update the relevant list?

Reply:
Same for remove issue (in that case, we could get rid of "isSender" and return the list directly)

Reply:
@larkox my issue is that I would need to return **two** arrays of lists (one for the sender/receiver and one for the initiator of the action). I don't really see the value in that.

Reply:
My idea is to return the list name, not the list array. That way, on the next line, instead of refreshing "my" and "in", we just refresh the meaningful list (as we are doing later for sender, where we just refresh "out").

Reply:
I don't know why it's so hard for me to understand this, but both lists (my and in) are "relevant". So I guess the return value needs to be an **array of list names**?